src/components/mentoring/posts/PostsSection.jsx

```diff
@@ -34,7 +33,7 @@ export default function PostsSection() {
         <h1 className="inline-block text-4xl font-bold text-green-700">
           Mentoring List
         </h1>
-        {isMentor && (
+        {data.data.response.role === "mentor" && (
```

Comment:
이런 상수 값들은 Enum 패턴으로 만들어보세요.
그리고 그렇게 만든 Enum 객체는 별도 모듈로 분리하고, 이를 참조해서 사용하세요.
오타로 인한 에러 발생이 크게 줄어들고, 자동완성이 가능해지며, 가독성이 좋아집니다.

```javascript
// consts/code.js
export default RoleType = {
  MENTOR: "MENTOR",
  ...
};

// SomeComponent.jsx
import { RoleType } from './consts/code';
```

```suggestion
        {data.data.response.role === RoleType.MENTOR && (
```